src/app/commands/cmd_goto_frame.cpp

```diff
@@ -88,6 +91,46 @@ class GotoNextFrameCommand : public GotoCommand {
   }
 };
 
+class GotoNextFrameWithSameTagCommand : public GotoCommand {
+public:
+  GotoNextFrameWithSameTagCommand() : GotoCommand("GotoNextFrameWithSameTag",
+                                                  "Go to Next Frame with same tag") { }
+  Command* clone() const override { return new GotoNextFrameWithSameTagCommand(*this); }
+
+protected:
+  frame_t onGetFrame(Editor* editor) override {
+    Sprite* sprite = editor->sprite();
+    frame_t currentFrame = editor->frame();
+    FrameTag* tag = get_animation_tag(sprite, currentFrame);
+    frame_t frameToGo = currentFrame + frame_t(1);
+
+    if (frameToGo > tag->toFrame())
```

Comment:
I didn't check this in runtime, but I think `tag` can be nullptr, so this line crashes if the command is used in animations without tags.